server/src/index.ts

```diff
@@ -104,6 +114,7 @@ const oauthLogin = (_: express.Request, res: express.Response) => {
 
 const oauthCallback = (req: express.Request, res: express.Response, next: express.NextFunction) => {
   const provider = opensrpAuth;
+  console.log("Original Url", req.originalUrl);
```

Comment:
Please remove this one when done.

Reply:
@p-netm curious why the lint test did not flag this one.  Do we need to create an issue to investigate this?

Reply:
Will create an issue.